iot-hub/Samples/service/RegistryManagerSample/RegistryManagerSample.cs

```diff
@@ -189,13 +189,13 @@ private async Task UpdateDesiredPropertiesAsync(RegistryManager registryManager,
 
             // Set a desired value for a property the device supports, with the corresponding data type
             var patch =
-@"{
-  ""properties"": {
-    ""desired"": {
-      ""customKey"": ""customValue""
-    }
-  }
-}";
+            @"{
+              ""properties"": {

```

Comment:
This will insert a bunch of additional spaces in the payload. Do we want to do that?

Reply:
Probably not but the sample works even with the spaces. I'll revert this.